test/ipns-pubsub.js

```diff
@@ -66,26 +70,23 @@ describe('ipns-pubsub', () => {
     return Promise.all(nodes.map((node) => node.stop()))
   })
 
-  it('should get enabled state of pubsub', function (done) {
-    nodes[0].api.name.pubsub.state((err, state) => {
-      expect(err).to.not.exist()
-      expect(state).to.exist()
-      expect(state.enabled).to.equal(true)
+  it('should get enabled state of pubsub', async function () {
+    const state = await nodes[0].api.name.pubsub.state()
 
-      done()
-    })
+    expect(state).to.exist()
+    expect(state.enabled).to.equal(true)
   })
 
-  it('should publish the received record to a go node and a js subscriber should receive it', function (done) {
+  it('should publish the received record to a go node and a js subscriber should receive it', async function () {
     this.timeout(300 * 1000)
 
-    subscribeToReceiveByPubsub(nodes[0], nodes[1], nodesIds[0].id, nodesIds[1].id, done)
+    await subscribeToReceiveByPubsub(nodes[0], nodes[1], nodesIds[0].id, nodesIds[1].id)
```

Comment:
return instead?

Reply:
I think `await` here is fine since there's no need to return a value from this function.